timeseries/predict.q

```diff
@@ -23,13 +26,18 @@ ts.AR.predict:{[model;exog;len]
 // @category modelPredict
 // @fileoverview Predictions based on an AutoRegressive Moving Average model 
 //   (ARMA)
+// @params config {dict} Information returned from `ml.ts.ARMA.fit`
+//   including:
+//   modelInfo - Model coefficients and data needed for future predictions
+//   predict - A projection allowing for prediction of future values
 // @param model {dict} Model parameters returned from fitting of an appropriate
```

Comment:
This can be removed as it's now incorporated within the config dictionary

Reply:
This is the case across the remainder of the predict functions in this file by the look of things